docs/run-documentation.sh

```diff
@@ -0,0 +1,10 @@
+python /wazuh-qa/deps/wazuh_testing/setup.py install
+cd /wazuh-qa
+export PYTHONPATH=/wazuh-qa/deps/wazuh_testing
+
+mkdir -p /var/ossec/queue/vulnerabilities/dictionaries/
+mkdir -p /var/ossec/etc/
```

Comment:
```suggestion
mkdir -p /var/ossec/queue/vulnerabilities/dictionaries/ /var/ossec/etc/
```

Reply:
Done in 618a8892f93dd794a6000baa838503371ec1eb6d